internal/definition/validate.go

```diff
@@ -93,3 +98,31 @@ func detectFiles(ctx context.Context, fpath string) ([]string, error) {
 
 	return defFiles, nil
 }
+
+// detect if the string is json
+func detectJson(msg string) bool {
```

Comment:
We have similar logic in a couple of places, perhaps a good candidate to place in the `internal/utils`?

```suggest
func IsJson(text string) bool
```

Reply:
Good point. I'm sure that can be accommodated.